modules/seqexec/engine/src/test/scala/seqexec/engine/SequenceSpec.scala

```diff
@@ -198,7 +199,7 @@ class SequenceSpec extends FlatSpec {
         )
       )
 
-    val c = ActionCoordsInSeq(1, ExecutionIndex(0), ActionIndex(1))
+    val c = ActionCoordsInSeq(1, GmosS, ExecutionIndex(0), ActionIndex(1))
```

Comment:
You forgot to revert this change

Reply:
Yeah, I didn't catch that with the `seqexec_server/test` command, which is what I have been using for testing. I think I need to execute a more general test command since things are slipping by.

That should be fixed now.